<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionEmploye</class>
 <widget class="QMainWindow" name="GestionEmploye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>931</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Ajouter Employe</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="conne">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>281</width>
        <height>238</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>nom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nom"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>prenom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="prenom"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>email</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="email"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>N Telephone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="telephone"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>CINE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="CINE"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>poste</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="poste"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="password"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>salaire</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="salaire"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Chercher Employe</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetEmploye_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>911</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID Client</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prenom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N Telephone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CNIE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>poste</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>salaire</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>password</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>441</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CNIE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="messagechercher"/>
       </item>
       <item>
        <widget class="QPushButton" name="chercher">
         <property name="text">
          <string>chercher</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Afficher Employe</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetEmploye">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>911</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID Client</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prenom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N Telephone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CNIE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>poste</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>salaire</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>password</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="ButtonMise">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Maise Ã  jour </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Supprimer Employe</string>
     </attribute>
     <widget class="QLabel" name="message_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>..</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>441</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CNIE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="messagesupprimer"/>
       </item>
       <item>
        <widget class="QPushButton" name="supprimerButton_2">
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Page d'accueillle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
